typescript-timers/typescript-timers-quiz-notes.md

```diff
@@ -0,0 +1,53 @@
+# typescript-timers-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is a "callback" function?
+
+a function that is passed as an argument to another function and then called by that function to complete a task
+
+- Besides adding an event listener callback function to an element or the `document`, what is one way to delay the execution of a JavaScript function until some point in the future?
+
+setTimeout{}
+
+- How can you set up a function to be called repeatedly without using a loop?
+
+an iterator
```

Comment:
The answer here is `setInterval` (since the exercise covered `setTimeout` and `setInterval`